requirements-dev.txt

```diff
@@ -1,9 +1,9 @@
 # Internationalization and Localization requirements
--e git://github.com/edx/xblock-sdk.git@v0.1.4#egg=xblock-sdk==v0.1.4
-Django==1.11.23
+-e git://github.com/edx/xblock-sdk.git@v0.1.7#egg=xblock-sdk==v0.1.7
+Django==2.2
```

Comment:
@kaizoku, this should pinned to the latest Django 2.2.x version (2.2.10 at the time of writing this) or at least use the `~=` specifier to always install the latest 2.2.x version.

Reply:
Ah right, I've taken both of those suggestions and set it to `Django~=2.2.10`